### PR TITLE
HOL 7 Update. WMS ID: 11155

### PR DESCRIPTION
**Pull request by tomohamm**
# WMS ID: 11155

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [x] All filenames are lower case (including folders, images, files, etc.)
- [x] Filenames are descriptive
- [x] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [x] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [x] Image references in markdown contain an alternate text

### CODE REVIEW

**Comment by tomohamm** [comment]
@hope-fisher Please review.

**Comment by hope-fisher** [comment]
@tomohamm please have the workshop owner add you as a part of the workshop team so that this and future PRs can be approved.

**Comment by tomohamm** [comment]
@hope-fisher I am added to the workshop team now. Please check.

**Review by hope-fisher — CHANGES_REQUESTED** [review]
all files must be in lowercase. Please update `apex-professional-hol7/images/Screenshot 2023-10-12 at 1.01.01 AM.png` accordingly.

File: apex-professional-hol7/workshops/tenancy/manifest.json
Comment (dannymgithub) [inline apex-professional-hol7/workshops/tenancy/manifest.json:1]: Just flagging that Lab1 appears twice below, in the event it causes issues due to duplication.

**Review by dannymgithub — CHANGES_REQUESTED** [review]
Just flagging that Lab1 appears twice below, in the event it causes issues due to duplication.

**Review by dannymgithub — APPROVED** [review]

**Review by dannymgithub — APPROVED** [review]

**Pull request merged**